packages/core/chrome/core-chrome-browser-internal/src/chrome_service.tsx

```diff
@@ -180,14 +180,21 @@ export class ChromeService {
     if (isDev) {
       setEuiDevProviderWarning((providerError) => {
         const errorObject = new Error(providerError.toString());
-        // show a stack trace in the console
+        // 1. show a stack trace in the console
         // eslint-disable-next-line no-console
         console.error(errorObject);
 
+        // 2. store error in sessionStorage so it can be detected in testing
+        sessionStorage.setItem('dev.euiProviderWarning.message', providerError.toString());
```

Comment:
tiny nit: I'd put all of this under a single key as a serialized json since they are part of the same state

Reply:
Good point. I have pushed 8d675a6f0eb